### DISCUSSION TITLE
Can performance on a small dataset become worse after fine-tuning?

### DISCUSSION THREAD

**Discussion by TPF2017** [amazon-science/chronos-forecasting#111] [Q&A] [answered]
After fine-tuning on a small data set, the performance on this data set is worse. Is this normal? What are the possible reasons?

**Comment by abdulfatir** [comment, marked as answer]
Hi @TPF2017!

Fine-tuning performance depends on a variety of factors, including hyperparameters such as the learning rate and training steps. If your dataset is small, it is possible that a specific selection of these hyperparameters (e.g., high learning rate or large number of step) worsens the performance. There are other potential causes such as distribution shifts in the time series.
